### PR TITLE
added middleware and began protecting alumni route

### PR DESCRIPTION
**Pull request by ojo88**

### CODE REVIEW

File: middleware/middleware.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = function(req, res, next) {
+ const token = req.header('x-auth-token');
+ if (!token) {
+    console.log('here');
```
Comment (zmays) [inline middleware/middleware.js]: usually you will want to remove logs like this before merging into master. Or, if you want to leave it, you could add some more detail, like: no auth token provided

File: routes/alumni.js [hunk 2]
```diff
@@ -4,10 +4,12 @@ var _ = require('lodash')
 const alumni = require('../models/alumni')
 require('../models/class')
 require('../models/language')
+const auth = require('../middleware/middleware')
 
 
 /* GET users listing. */
 router.get('/', function(req, res, next) {
+  console.log(req)
```
Comment (zmays) [inline routes/alumni.js]: same with this log, probably will want to remove it

File: routes/alumni.js [hunk 3]
```diff
@@ -4,10 +4,12 @@ var _ = require('lodash')
 const alumni = require('../models/alumni')
 require('../models/class')
 require('../models/language')
+const auth = require('../middleware/middleware')
```
Comment (zmays) [inline routes/alumni.js]: any reason for importing this middleware?

File: routes/class.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ router.get('/', function (req, res, next) {
         })
 });
 
-router.post('/', function (req, res, next) {
+router.post('/', auth, function (req, res, next) {
```
Comment (zmays) [inline routes/class.js:20]: nice work

File: routes/user.js [hunk 5]
```diff
@@ -16,9 +16,10 @@ router.post('/login', function (req, res, next) {
 				res.status(404).send('screw off')
 			} else {
 				console.log(matchedUser)
```
Comment (zmays) [inline routes/user.js]: probs don't want to log this here as well

**Review by zmays — APPROVED** [review]
approved, just make sure to move those logs

File: routes/alumni.js (hunk 3 above)
Comment (ojo88) [reply routes/alumni.js]: There is no reason for the middleware to be there. Was just there when I was testing the middleware. Has been removed.

**Pull request merged**